### PR TITLE
Readme updated

### PR DESCRIPTION
**Pull request by juanmanuelramallo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,61 @@
...
 
-_TODO_
+```
+rsgem new NAME [CI_PROVIDER]
+```
+
+RSGem will solve the following tasks for you:
+
+1. Create a folder for your gem leveraging bundler's defaults. (You need bundler in your system)
+1. Add the following dependencies:
```
Comment (santib) [inline README.md:25]: why all `1.`s instead of 1. 2. 3. etc?

File: README.md [hunk 2]
```diff
@@ -8,16 +8,61 @@
...
+    - [Rake](https://github.com/ruby/rake)
+    - [Reek](https://github.com/troessner/reek)
+    - [RSpec](https://github.com/rspec/rspec)
+    - [Rubocop](https://github.com/rubocop-hq/rubocop)
+    - [Simplecov](https://github.com/colszowka/simplecov)
+        - We use Simplecov at 0.17.1 because that's the latest compatible version with CodeClimate.
+1. Add configuration files for Reek and Rubocop with default Rootstrap's configuration.
+1. Add a rake task to run Rubocop and Reek by calling `rake code_analysis`.
```
Comment (santib) [inline README.md:33]:
```suggestion
1. Add a rake task to run Rubocop and Reek by calling `bundle exec rails code_analysis`.
```

File: README.md [hunk 3]
```diff
@@ -8,16 +8,61 @@
...
+1. [Clean the Gemfile](https://github.com/rootstrap/tech-guides/blob/master/open-source/developing_gems.md#gemfilegemfilelockgemspec).
+1. [Git ignore the Gemfile.lock](https://github.com/rootstrap/tech-guides/blob/master/open-source/developing_gems.md#gemfilegemfilelockgemspec)
+1. Add a CI provider configuration. GitHub Actions and Travis are available providers. GitHub Actions is the default.
```
Comment (santib) [inline README.md]: 😏 I thought we said to keep Travis as the default 😛

**Review by santib — APPROVED** [review]
This is amazing, great job. Approved after comments

File: README.md (hunk 1 above)
Comment (juanmanuelramallo) [reply README.md:25]: The markdown parser converts them to the correct number

File: README.md (hunk 2 above)
Comment (juanmanuelramallo) [reply README.md:33]: This is no rails, but we can add the bundle exec part as a good practice maybe

File: README.md (hunk 3 above)
Comment (juanmanuelramallo) [reply README.md]: Yeah! My bad, will update

**Pull request merged**